.github/bug_report.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Bug report
+about: Create a bug report to help us improve QGIS.
+title: ''
+labels: ''
+assignees: ''
+
+---
```

Comment:
What is this first section above? Do we need to fill manually labels and assignees?
Do we need to keep `''`?
Sorry if I missed something

Reply:
Hi, the first section is the metadata section for github's template where you can put defaults title, labels and assignments. I'all add the default label 'bug report' at least.